src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -0,0 +1,81 @@
+        String organisationIdentifier = (String) response.get("organisationIdentifier");
+        assertThat(organisationIdentifier).isNotEmpty();
+        professionalApiClient.updateOrganisation(organisationIdentifier);
+
+        Map<String, Object> searchResponse = professionalApiClient.searchUsersByOrganisation(organisationIdentifier, "False", HttpStatus.OK);
+
+        assertThat(searchResponse.get("users")).asList().isNotEmpty();
+
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+        HashMap professionalUsersResponse = professionalUsersResponses.get(0);
+
+        assertThat(professionalUsersResponse.get("firstName")).isNotNull();
+        assertThat(professionalUsersResponse.get("lastName")).isNotNull();
+        assertThat(professionalUsersResponse.get("email")).isNotNull();
+        assertThat(professionalUsersResponse.get("status")).isNotNull();
+        assertThat(((List)professionalUsersResponse.get("roles")).size()).isEqualTo(0);
+    }
+
+    @Test
+    public void find_users_by_active_organisation_with_showDeleted_True() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String organisationIdentifier = (String) response.get("organisationIdentifier");
+        assertThat(organisationIdentifier).isNotEmpty();
+        professionalApiClient.updateOrganisation(organisationIdentifier);
+
+        Map<String, Object> searchResponse = professionalApiClient.searchUsersByOrganisation(organisationIdentifier, "False", HttpStatus.OK);
+
+        assertThat(searchResponse.get("users")).asList().isNotEmpty();
+
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+        HashMap professionalUsersResponse = professionalUsersResponses.get(0);
+
+        assertThat(professionalUsersResponse.get("firstName")).isNotNull();
+        assertThat(professionalUsersResponse.get("lastName")).isNotNull();
+        assertThat(professionalUsersResponse.get("email")).isNotNull();
+        assertThat(professionalUsersResponse.get("status")).isNotNull();
+        assertThat(((List)professionalUsersResponse.get("roles")).size()).isEqualTo(0);
+    }
+
+    @Test
+    public void find_users_for_non_active_organisation() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String organisationIdentifier = (String) response.get("organisationIdentifier");
+        assertThat(organisationIdentifier).isNotEmpty();
+
+        Map<String, Object> searchResponse = professionalApiClient.searchUsersByOrganisation(organisationIdentifier, "False", HttpStatus.OK);
+
+        assertThat(searchResponse.get("users")).asList().isEmpty();
+    }
+
+    @Test
+    public void find_users_for_non_existing_organisation() {
+
+        professionalApiClient.searchUsersByOrganisation(UUID.randomUUID().toString(), "False", HttpStatus.NOT_FOUND);
+

```

Comment:
could you please add another test if we pass showDeleted="invalidentry"

Reply:
Added one more functional test